pandas/core/indexes/base.py

```diff
@@ -3112,8 +3112,11 @@ def map(self, mapper, na_action=None):
 
     def isin(self, values, level=None):
         """
+        Boolean array on existence of index values in `values`.
```

Comment:
Does

    Return a boolean array where the index values are in `values`.

fit on a line?